Beginning of a log file: "Fascist Tweet Archiver".
System info: posix.uname_result(sysname='Linux', nodename='linawesome', release='4.9.0-3-amd64', version='#1 SMP Debian 4.9.30-2+deb9u3 (2017-08-06)', machine='x86_64')
Log started 2017-09-12 03:34:57.993954



INFO: read target account data; we're tracking 45 accounts
WARNING: no ProtocolError (got exception "name 'ProtocolError' is not defined"); trying requests.packages.urllib3.exceptions instead
NOTE: successfully imported from requests
